### PR TITLE
fix(agw): updated debian repo for 1.8.0 release

### PR DESCRIPTION
**Pull request by ShubhamTatvamasi**
Signed-off-by: Shubham Tatvamasi <shubhamtatvamasi@gmail.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: cwf/gateway/docker/go/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN apt-get update && \
     apt-get install -y apt-utils software-properties-common apt-transport-https
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main"
+    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-ci main"
```
Comment (maxhbr) [inline cwf/gateway/docker/go/Dockerfile]:
```suggestion
    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-ci focal-1.8.0"
```

File: cwf/gateway/docker/python/Dockerfile [hunk 2]
```diff
@@ -49,7 +49,7 @@ COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 COPY cwf/gateway/deploy/roles/ovs/files/magma-preferences /etc/apt/preferences.d/
 # TODO: change repo to focal once there will be such
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main" && \
+    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-ci main" && \
```
Comment (maxhbr) [inline cwf/gateway/docker/python/Dockerfile]:
```suggestion
    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-1.8.0 main" && \
```

File: feg/gateway/docker/go/Dockerfile [hunk 3]
```diff
@@ -9,7 +9,7 @@ RUN apt-get update && \
     apt-get install -y apt-utils software-properties-common apt-transport-https
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main"
+    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-ci main"
```
Comment (maxhbr) [inline feg/gateway/docker/go/Dockerfile]:
```suggestion
    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-1.8.0 main"
```

File: feg/gateway/docker/python/Dockerfile [hunk 4]
```diff
@@ -68,7 +68,7 @@ RUN apt-get update && \
     apt-get install -y apt-utils software-properties-common apt-transport-https curl
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main"
+    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-ci main"
```
Comment (maxhbr) [inline feg/gateway/docker/python/Dockerfile]:
```suggestion
    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-1.8.0 main"
```

File: orc8r/gateway/docker/Dockerfile [hunk 5]
```diff
@@ -41,7 +41,7 @@ RUN apt-get update && \
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 COPY cwf/gateway/deploy/roles/ovs/files/magma-preferences /etc/apt/preferences.d/
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main"
+    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-ci main"
```
Comment (maxhbr) [inline orc8r/gateway/docker/Dockerfile]:
```suggestion
    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-1.8.0 main"
```

Comment (maxhbr) [reply orc8r/gateway/docker/Dockerfile]: this one is still open

File: orc8r/gateway/docker/Dockerfile [hunk 6]
```diff
@@ -41,7 +41,7 @@ RUN apt-get update && \
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
 COPY cwf/gateway/deploy/roles/ovs/files/magma-preferences /etc/apt/preferences.d/
 RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main"
+    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian focal-1.8.0 main"
```
Comment (ShubhamTatvamasi) [inline orc8r/gateway/docker/Dockerfile:44]: @maxhbr I have updated this.

File: orc8r/gateway/docker/Dockerfile (hunk 5 above)
Comment (maxhbr) [reply orc8r/gateway/docker/Dockerfile]: sorry, not sure why it is still not listed as outdated. Let me resolve it.

**Comment by m-govind** [comment]
Looks good to me

**Comment by ShubhamTatvamasi** [comment]
> changes to `hil_testing/Magma_Ansible/project/roles/upgrade-prep/tasks/main.yaml` are not necessary, it is unused code

Reverted this change.

**Comment by m-govind** [comment]
Looks good to me

**Comment by ShubhamTatvamasi** [comment]
> On this branch the build for the gateway_c image failed: https://github.com/magma/magma/runs/8157992438?check_suite_focus=true
> 
> 

Build is fixed now.

**Review by m-govind — APPROVED** [review]

**Comment by jheidbrink** [comment]
When building the `agw_gateway_python` container on this branch with an ARM instance, `apt-get` fails with

**Comment by sebathomas** [comment]
Strange, shouldn't it be this one?
https://artifactory.magmacore.org/ui/repos/tree/General/debian-prod%2Ffocal-1.8.0%2Fbcc-tools_0.23-1_arm64.deb

**Comment by maxhbr** [comment]
> Strange, shouldn't it be this one? https://artifactory.magmacore.org/ui/repos/tree/General/debian-prod%2Ffocal-1.8.0%2Fbcc-tools_0.23-1_arm64.deb

it is missing relevant metadata in artifactory, also in `debian-test`.

**Comment by LKreutzer** [comment]
> When building the `agw_gateway_python` container on this branch with an ARM instance, `apt-get` fails with
> 
> > Unable to locate package bcc-tools

bcc-tools for arm64 had damaged property values for debian-test and debian. Removed respective files and re-uploaded with correct properties.

@jheidbrink please test again

**Comment by jheidbrink** [comment]
Looks good so far. The `bazel build` is still running, but `bcc-tools` installed fine.

**Comment by crasu** [comment]
@ShubhamTatvamasi do you have a green feg/lte integ test for this pr?

**Comment by ShubhamTatvamasi** [comment]
> @ShubhamTatvamasi do you have a green feg/lte integ test for this pr?

I don't have any data on those tests.

**Comment by sebathomas** [comment]
You can run them on the branch in your fork if you enable Github Actions.

**Comment by ShubhamTatvamasi** [comment]
> You can run them on the branch in your fork if you enable Github Actions.

Triggered both tests.

LTE integ test:
https://github.com/ShubhamTatvamasi/magma/actions/runs/3005169584

Federated integ test:
https://github.com/ShubhamTatvamasi/magma/actions/runs/3005170027

**Comment by sebathomas** [comment]
The LTE Integ Test looks good, I think the Federated Integ Test shows some problems in the agw-orc8r connection. Does anyone have an idea if it was green on v1.8 before? (Github seems to have some issues at the moment, hard to find out in the UI...)

**Comment by crasu** [comment]
> The LTE Integ Test looks good, I think the Federated Integ Test shows some problems in the agw-orc8r connection. Does anyone have an idea if it was green on v1.8 before? (Github seems to have some issues at the moment, hard to find out in the UI...)

Yepp the LTE Integ Test only failed because it cannot upload its state to the website
@ShubhamTatvamasi Could you give the federated integ test another try?

**Comment by ShubhamTatvamasi** [comment]
> 

https://github.com/ShubhamTatvamasi/magma/actions/runs/3009501470

**Pull request merged**